### DISCUSSION TITLE
Recording on sub channel

### DISCUSSION THREAD

**Discussion by jmntn2000** [blakeblackshear/frigate#3445] [Ask A Question] [answered]
I finally got around to upgrading to version .10.  I have my config file working but I don't see how to get back previous functionality of 24/7 recording of sub stream for smaller file sizes and clip recordings in full HD.  Before I could put roll of sub stream to record and role of full HD to clips.  How do I get back to this in .10 where clips is removed and everything configured under a single role of record?

**Comment by NickM-27** [comment, marked as answer]
In previous versions there was a lot of redundant storage where the same clip could be stored in multiple places.

In 0.10+ the `record` role saves recordings in 10 second segments and they are joined together to build recordings and events clips which saves considerable space. This does mean however that if you want your clips to be full resolution then you'll need the 24/7 recording to be full resolution as well. 

There are [different retention modes](https://docs.frigate.video/configuration/record#what-do-the-different-retain-modes-mean) to only save segments of interest and reduce overall storage used. There is also a [pinned feature request](https://github.com/blakeblackshear/frigate/issues/2652) for having both streams recorded and switching between them when motion is detected / active_objects etc.

**Reply by jmntn2000** [reply]
Thank you, that makes sense.
